pkg/sdn/plugin/bin/openshift-sdn-ovs

```diff
@@ -69,10 +69,14 @@ add_ovs_flows() {
 }
 
 del_ovs_flows() {
-    ovs-ofctl -O OpenFlow13 del-flows br0 "ip,nw_dst=${ipaddr}"
-    ovs-ofctl -O OpenFlow13 del-flows br0 "ip,nw_src=${ipaddr}"
-    ovs-ofctl -O OpenFlow13 del-flows br0 "arp,nw_dst=${ipaddr}"
-    ovs-ofctl -O OpenFlow13 del-flows br0 "arp,nw_src=${ipaddr}"
+    if [ "$ipaddr" != "" ]; then
+        ovs-ofctl -O OpenFlow13 del-flows br0 "ip,nw_dst=${ipaddr}"
+        ovs-ofctl -O OpenFlow13 del-flows br0 "ip,nw_src=${ipaddr}"
+        ovs-ofctl -O OpenFlow13 del-flows br0 "arp,nw_dst=${ipaddr}"
+        ovs-ofctl -O OpenFlow13 del-flows br0 "arp,nw_src=${ipaddr}"
+    else
+        echo "No ip address given for teardown of ${veth_host}, could have been called twice.  Skipping flow deletion"
```

Comment:
I'll pull it.  The only reason I added it was if we thought we could get called the first time with no IP... but that would probably error all over the place if it happened.